### ISSUE TITLE
real-time pose tracking

### ISSUE THREAD

**Issue by week0425** [wenbowen123/BundleTrack#35]
Hello, I am studying the problems related to the recognition and grasping of objects by mechanical arms, so I want to know whether this project can realize real-time pose tracking. If not, is there any way to modify it to achieve real-time pose tracking?

**Comment by wenbowen123** [comment]
Yes, it can run about 10hz. [Watch this video](https://www.youtube.com/watch?v=WAr8ZY3mYyw) for an example using this for real-time closed-loop feedback manipulation.

**Comment by week0425** [comment]
I see. Thanks for your reply.
